autogen/README.md

```diff
@@ -182,10 +182,9 @@ To more cleanly handle cases where desired functionality would require complex d
 
 The root module is generated by running `make generate`. Changes to this repository should be made in the [`autogen`](/autogen) directory where appropriate.
 
-Note: The correct sequence to update the repo using autogen
-functionality is the run `make generate && make generate_docs`.  This
-will create the various Terraform files, and then generate the
-Terraform documentation using `terraform-docs`.
+Note: The correct sequence to update the repo using autogen functionality is the run 
```

Comment:
```suggestion
Note: The correct sequence to update the repo using autogen functionality is to run 
```

Reply:
@aaron-lane just fixed it :)

Reply:
Doesn't look fixed?

Reply:
Now that I have not messed up git and rebasing ... https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/191/files#diff-34a5f0b5ab84cf5f33367f7912231babR185 

Now it is fixed